servers/rendering/renderer_rd/storage_rd/particles_storage.cpp

```diff
@@ -1615,7 +1622,7 @@ void ParticlesStorage::update_particles() {
 			RD::get_singleton()->compute_list_bind_uniform_set(compute_list, particles->trail_bind_pose_uniform_set, 2);
 			RD::get_singleton()->compute_list_set_push_constant(compute_list, &copy_push_constant, sizeof(ParticlesShader::CopyPushConstant));
 
-			RD::get_singleton()->compute_list_dispatch_threads(compute_list, total_amount, 1, 1);
+			RD::get_singleton()->compute_list_dispatch_threads(compute_list, (uint32_t)ceil((float)total_amount / 64.0), 1, 1);
```

Comment:
Magic 64?

Reply:
Already marked as a bug (see my [comment](https://github.com/godotengine/godot/pull/90284#issuecomment-2054173888) "Particle FX compute shader").

That must not be merged.